src/util/invariant.h

```diff
@@ -202,19 +202,21 @@ inline void invariant_violated_string(
 #define __this_function__ __func__
 #endif
 
-#define INVARIANT(CONDITION, REASON) \
-  do /* NOLINT */ \
-  { \
-  if(!(CONDITION)) \
-    invariant_violated_string(__FILE__, __this_function__, __LINE__, (REASON)); /* NOLINT */  \
-  } while(0)
+#define INVARIANT(CONDITION, REASON)                                           \
+  do /* NOLINT */                                                              \
+  {                                                                            \
+    if(!(CONDITION))                                                           \
+      invariant_violated_string(                                               \
+        __FILE__, __this_function__, __LINE__, (REASON)); /* NOLINT */         \
+  } while(false)
```

Comment:
1) Which are these "some" warnings? 2) The previous code even worked within C code, while this is now C++; is that really necessary? 3) Please place this commit in a separate PR.

Reply:
1. CLion warnings when INVARIANT is used 2. `false` is already used in this file, so I don't think this changes anything. 3. done here: https://github.com/diffblue/cbmc/pull/2078